instrumentation-tests/build.gradle

```diff
@@ -14,7 +14,7 @@ android {
 
     defaultConfig {
         applicationId "com.mapbox.navigation.instrumentation_tests"
-        minSdkVersion androidVersions.minSdkVersion
+        minSdkVersion androidVersions.carMinSdkVersion
```

Comment:
I don't think we want to do this 🤔 . Also, supporting multiple minSdkVersions is difficult inside a single app

Reply:
If we do this, we still have a line of defense with the `:examples:` app which is used with the `run-firebase-robo:`. That path is also being deprecated. So we are running out of minSdkVersion 21 testing solutions

Reply:
```
              --device model=hammerhead,version=23,locale=fr,orientation=portrait \
              --device model=oriole,version=31,locale=de,orientation=landscape \
              --device model=redfin,version=30,locale=en,orientation=portrait \
```

actually we're not testing 21 anyways 😄